src/components/EventForm/EventForm.tsx

```diff
@@ -77,8 +79,14 @@ function EventForm({ onSubmit, ...rest }: EventFormProps) {
 
   const parseDate = (date: string, time: string) => new Date(date + 'T' + time);
 
-  const isFormValid = name && description && eventType && startTime && endTime &&
-    parseDate(date, startTime) > now && parseDate(date, endTime) > now;
+  const isFormValid =
+    name &&
+    description &&
+    eventType &&
+    startTime &&
+    endTime &&
+    parseDate(date, startTime) > now &&
+    parseDate(date, endTime) > now;
```

Comment:
If the form gets any more complex than this, then it might be a good idea to introduce Formik and Yup.

Reply:
I tried adding some form validation, and I agree that introducing Formik and Yup would be beneficial.